clusters/app.ci/assets/dptp-controller-manager.yaml

```diff
@@ -139,7 +139,14 @@ spec:
         - --registrySyncerOptions.image-stream-namespace=integr8ly
         - --registrySyncerOptions.image-stream-prefix=openshift/knative-
         - --registrySyncerOptions.image-stream-prefix=openshift/kubefed-operator-
+        # TODO (hongkliu): sync the whole namespace when ready
+        #- --registrySyncerOptions.image-stream-namespace=ocp
+        #- --registrySyncerOptions.image-stream-namespace=origin
         - --registrySyncerOptions.image-stream-prefix=ocp/ocs-
+        - --registrySyncerOptions.image-stream-tag=ocp/4.8:openstack-cinder-csi-driver-operator
+        - --registrySyncerOptions.image-stream-tag=ocp/4.8:operator-marketplace
+        - --registrySyncerOptions.image-stream-prefix=origin/4.
```

Comment:
We could copy this over 
https://github.com/openshift/release/blob/d593f1b21adf4de4fe44a68902b52bc85eca8496/core-services/release-controller/admin_deploy-origin-controller.yaml#L49

But there are lots of them in the folder.
Let us do them all together.

Currently, we have all those applied on api.ci already (by a previous mistake).